test/pretty.test.js

```diff
@@ -83,6 +83,24 @@ test('can send pretty print to custom stream', async ({ is }) => {
   log.info('foo')
 })
 
+test('multi-stream with pretty print', async ({ doesNotThrow }) => {
+  doesNotThrow(() => {
+    const { multistream } = require('pino-multi-stream')
+    const outStream = pino({
+      prettifier: require('pino-pretty'),
+      prettyPrint: {
+        levelFirst: true,
+        colorize: false
+      }
+    })
+    const log = pino({}, multistream([
+      { stream: outStream[pino.symbols.streamSym] }
+    ]))
+
+    log.info('foo')
```

Comment:
This is writing a log line to `stdout` when running the tests. Test output should be silent so that it does not interfere with the test runner's output.